templates/default/hadoop-init.erb

```diff
@@ -73,7 +73,12 @@ CONF_DIR="<%= @options['confdir'] %>"
 PID_FILE="<%= @options['pidfile'] %>"
 EXE_FILE="<%= @options['binary'] %>"
 EXE_ARGS="<%= @options['args'] %>"
+<% if node['platform_family'] == 'rhel' %>
 LOCKFILE="/var/lock/subsys/<%= @options['name'] %>"
+<% else %>
```

Comment:
Do we have something in place to prevent another platform/OS (besides rhel/debian) from getting installed (and breaking this)?
Or is that just not possible?